sdk/loadtestservice/Azure.Developer.LoadTesting/README.md

```diff
@@ -0,0 +1,135 @@
+# Azure Load Testing client library for .NET
+Azure Load Testing provides client library in .NET to the user by which they can interact natively with Azure Load Testing service. Azure Load Testing is a fully managed load-testing service that enables you to generate high-scale load. The service simulates traffic for your applications, regardless of where they're hosted. Developers, testers, and quality assurance (QA) engineers can use it to optimize application performance, scalability, or capacity.
+
+  [Source code](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/loadtestservice/Azure.Developer.LoadTesting/src) | [Package (NuGet)](https://www.nuget.org/packages?q=Azure.Developer.Loadtesting) | [API reference documentation](https://azure.github.io/azure-sdk-for-net) | [Product documentation](https://learn.microsoft.com/azure/load-testing/)
+
+
+## Documentation
+
+Various documentation is available to help you get started
+
+- [API reference documentation][api_reference_doc]
+- [Product Documentation][product_documentation]
+ 
```

Comment:
@NiveditJain these links may not work. Can we please update?

Reply:
@ninallam these links point to, 
[api_reference_doc]: https://docs.microsoft.com/rest/api/loadtesting/
[product_documentation]: https://azure.microsoft.com/services/load-testing/

Please let me know if you have some other link(s).